RECIPE: 31 plant based recipes

oakes777 opened the pull request:
This URL takes one to a lovely webpage with 31 delicious recipes which are all plant-based!

auberonedu commented:
Looks great. Some newer changes have caused conflicts though. Please pull and resolve the conflicts, then push again. Thanks!

oakes777 commented:
Hi Auberon,
I went back in and pulled/pushed but as I kept checking git status I could find no conflicts. Here’s the record:
--------------------------------------------------------

(base) ***@***.*** hot-links % ls
CODE_OF_CONDUCT.md      CONTRIBUTING.md         LICENSE                 README.md               pages
(base) ***@***.*** hot-links % git pull
Already up to date.
(base) ***@***.*** hot-links % git status
On branch recipeBranch
Your branch is up to date with 'origin/recipeBranch'.

nothing to commit, working tree clean
(base) ***@***.*** hot-links % git switch artBranch
Switched to branch 'artBranch'
Your branch is up to date with 'origin/artBranch'.
(base) ***@***.*** hot-links % git pull
Already up to date.
(base) ***@***.*** hot-links % git status
On branch artBranch
Your branch is up to date with 'origin/artBranch'.

nothing to commit, working tree clean
(base) ***@***.*** hot-links % git push --set-upstream origin artBranch   
branch 'artBranch' set up to track 'origin/artBranch'.
Everything up-to-date
(base) ***@***.*** hot-links % git switch main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(base) ***@***.*** hot-links % git switch recipeBranch
Switched to branch 'recipeBranch'
Your branch is up to date with 'origin/recipeBranch'.
(base) ***@***.*** hot-links % git status
On branch recipeBranch
Your branch is up to date with 'origin/recipeBranch'.

nothing to commit, working tree clean
(base) ***@***.*** hot-links % git push --set-upstream origin recipeBranch
branch 'recipeBranch' set up to track 'origin/recipeBranch'.
Everything up-to-date
(base) ***@***.*** hot-links % git status
On branch recipeBranch
Your branch is up to date with 'origin/recipeBranch'.

nothing to commit, working tree clean
(base) ***@***.*** hot-links % git switch main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(base) ***@***.*** hot-links % git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
(base) ***@***.*** hot-links % 
————————————————————————————
I’m not sure what I’m doing wrong, please advise.
Thanks,
Jonathan

> On Nov 7, 2024, at 16:26, Auberon López ***@***.***> wrote:
> 
> 
> Looks great. Some newer changes have caused conflicts though. Please pull and resolve the conflicts, then push again. Thanks!
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/auberonedu/hot-links/pull/20#issuecomment-2463482634>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFD6K6IF3UCP4WNMH5O6OTDZ7QAMFAVCNFSM6AAAAABRMIRXDKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINRTGQ4DENRTGQ>.
> You are receiving this because you authored the thread.
>

oakes777 commented:
Ok, I figured out what I needed to do - I had to resolve the two conflicts on GitHub.com first, then go back to terminal and pull then push origin for each branch. Seems both are resolved and in the green currently. Here's my terminal log:


(base) mb16@mb16 hot-links % git pull origin recipeBranch
remote: Enumerating objects: 23, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 23 (delta 10), reused 9 (delta 6), pack-reused 5 (from 1)
Unpacking objects: 100% (23/23), 5.93 KiB | 319.00 KiB/s, done.
From https://github.com/oakes777/hot-links
 * branch            recipeBranch -> FETCH_HEAD
   2ce003d..85b112c  recipeBranch -> origin/recipeBranch
Updating 2ce003d..85b112c
Fast-forward
 pages/art.md     | 5 +++++
 pages/coding.md  | 7 ++++++-
 pages/recipes.md | 5 +++++
 3 files changed, 16 insertions(+), 1 deletion(-)
(base) mb16@mb16 hot-links % git push origin recipeBranch
Everything up-to-date
(base) mb16@mb16 hot-links % git switch main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(base) mb16@mb16 hot-links % git switch artBranch
Switched to branch 'artBranch'
Your branch is up to date with 'origin/artBranch'.
(base) mb16@mb16 hot-links % git pull origin artBranch   
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4 (delta 0), reused 3 (delta 0), pack-reused 0 (from 0)
Unpacking objects: 100% (4/4), 1.54 KiB | 394.00 KiB/s, done.
From https://github.com/oakes777/hot-links
 * branch            artBranch  -> FETCH_HEAD
   2795ce5..4ee5a33  artBranch  -> origin/artBranch
Updating 2795ce5..4ee5a33
Fast-forward
 pages/art.md     | 5 +++++
 pages/coding.md  | 7 ++++++-
 pages/recipes.md | 7 ++++++-
 3 files changed, 17 insertions(+), 2 deletions(-)
(base) mb16@mb16 hot-links % git push origin artBranch   
Everything up-to-date
(base) mb16@mb16 hot-links %

oakes777 commented:
Ok, I think I figured out what to do and the order to do it in :^))
I had to resolve the two branch conflicts on GitHub.com first and ‘commit merge’ then go back into terminal and pull/push origin of each branch. Seems this is resolved currently, please advise if I need to do anything else, thanks. Here’s the terminal log:


(base) ***@***.*** hot-links % git pull origin recipeBranch
remote: Enumerating objects: 23, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 23 (delta 10), reused 9 (delta 6), pack-reused 5 (from 1)
Unpacking objects: 100% (23/23), 5.93 KiB | 319.00 KiB/s, done.
From https://github.com/oakes777/hot-links
 * branch            recipeBranch -> FETCH_HEAD
   2ce003d..85b112c  recipeBranch -> origin/recipeBranch
Updating 2ce003d..85b112c
Fast-forward
 pages/art.md     | 5 +++++
 pages/coding.md  | 7 ++++++-
 pages/recipes.md | 5 +++++
 3 files changed, 16 insertions(+), 1 deletion(-)
(base) ***@***.*** hot-links % git push origin recipeBranch
Everything up-to-date
(base) ***@***.*** hot-links % git switch main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(base) ***@***.*** hot-links % git switch artBranch
Switched to branch 'artBranch'
Your branch is up to date with 'origin/artBranch'.
(base) ***@***.*** hot-links % git pull origin artBranch   
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4 (delta 0), reused 3 (delta 0), pack-reused 0 (from 0)
Unpacking objects: 100% (4/4), 1.54 KiB | 394.00 KiB/s, done.
From https://github.com/oakes777/hot-links
 * branch            artBranch  -> FETCH_HEAD
   2795ce5..4ee5a33  artBranch  -> origin/artBranch
Updating 2795ce5..4ee5a33
Fast-forward
 pages/art.md     | 5 +++++
 pages/coding.md  | 7 ++++++-
 pages/recipes.md | 7 ++++++-
 3 files changed, 17 insertions(+), 2 deletions(-)
(base) ***@***.*** hot-links % git push origin artBranch   
Everything up-to-date
(base) ***@***.*** hot-links % 

Cheers
J.

> On Nov 7, 2024, at 16:26, Auberon López ***@***.***> wrote:
> 
> 
> Looks great. Some newer changes have caused conflicts though. Please pull and resolve the conflicts, then push again. Thanks!
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/auberonedu/hot-links/pull/20#issuecomment-2463482634>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BFD6K6IF3UCP4WNMH5O6OTDZ7QAMFAVCNFSM6AAAAABRMIRXDKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINRTGQ4DENRTGQ>.
> You are receiving this because you authored the thread.
>
